portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <header>
+        <div id ="header">
+            <h1 style="text-align:center"> My Projects </h1>
+            <div id="padding">
+                <a href="index.html"> Home </a>
+            </div>
+            <div id="padding">
+                <a href="projects.html"> Projects </a>
+            </div>
+        </div>
+  </header>
+    <body>   
+        <div class="img-container" style="width:400px">
```

Comment:
For these and for images, move the styles to CSS.

You can do something like:

.img-container {
  width: 400px;
}

.img-container img {
  width: 300px;
  height: 400px;
}

.img-container img is a selector meaning "all < img > elements that are a descendant of the img-container" class

Reply:
solved